dune-project

```diff
@@ -1 +1,46 @@
-(lang dune 2.6)
\ No newline at end of file
+(lang dune 2.8)
+
+; Warning: The flag set for these foreign sources overrides the `:standard` set
+; of flags. However the flags in this standard set are still added to the
+; compiler arguments by Dune. This might cause unexpected issues. You can
+; disable this warning by defining the option `(use_standard_c_and_cxx_flags <bool>)` 
+; in your `dune-project` file. Setting this option to `true` will
+; effectively prevent Dune from silently adding c-flags to the compiler
+; arguments which is the new recommended behaviour.
+(use_standard_c_and_cxx_flags false)
+
+(generate_opam_files true)
+
+(source (github dmtrKovalenko/odiff))
+(license MIT)
+(authors "Dmitriy Kovalenko")
+(maintainers "https://dmtrkovalenko.dev" "dmtr.kovalenko@outlook.com")
+
+(package
+ (name odiff)
+ (depends
+  odiff-core
+  odiff-io
+ )
+)
+
+(package
+ (name odiff-core)
+ (depends
+  dune
+  (reason (and (>= 3.6.0) (< 4.0.0)))
+  (ocaml (and (>= 4.10.0) (< 5.0.0)))
+ )
+)
+
+(package
+ (name odiff-io)
+ (depends
+  dune
+  conf-libpng
+  odiff-core
+  (camlimages (= 5.4.0))
+  (reason (and (>= 3.6.0) (< 4.0.0)))
+  (ocaml (and (>= 4.10.0) (< 4.11.0)))
+ )
+)
```

Comment:
Which synopsis / description should we use for odiff-core and odiff-io?
Can you add one or tell me, which one you want to have in there? 
The one used in the package.json doesn't really fit in my opinion.

Reply:
Probably "Pixel-by-pixel image difference algorithm compatible with camlimages"?